web/src/components/common/CommitProgress.tsx

```diff
@@ -39,8 +40,10 @@ const CommitProgressContainer = styled.div`
         return 'column';
     }
   }};
+
   text-align: right;
-  font-size: 0.8em;
+  font-size: ${({largerFont}: CommitProgressContainerProps) =>
+    largerFont ? '1em' : '0.8em'};
```

Comment:
In what situations are you using largerFont and normal font?

Reply:
@xlai20 Ah I will use largerFont in a full details page, and normal font in listings cards.